client/lib/analytics/ad-tracking.js

```diff
@@ -240,6 +240,32 @@ function retarget() {
 }
 
 /**
+ * Fire ad tracking events for selected vertical
+ *
+ * @param {String} vertical - The name of the selected vertical
+ * @returns {void}
+ */
+function recordSelectedVertical( vertical ) {
+	// Facebook
+	if ( window.fbq ) {
+		window.fbq(
+			'trackCustom',
+			'VerticalSelectedEvent',
+			{ vertical }
+		);
+	}
+
+	// Adwords
+	if ( window.google_trackConversion ) {
+		window.google_trackConversion( {
+			google_conversion_id: GOOGLE_CONVERSION_ID,
+			google_custom_params: { vertical },
+			google_remarketing_only: true
+		} );
```

Comment:
it looks like these might be fundamental function calls with specific values instead of specific functions which get called from the outside. do they really need to exists here in `lib/analytics` or if we're just using the underlying functions could we maybe just leave them in the new action handler file?